client/my-sites/stats/wordads/index.jsx

```diff
@@ -0,0 +1,192 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import page from 'page';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, numberFormat } from 'i18n-calypso';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import StatsNavigation from 'blocks/stats-navigation';
+import StatsPeriodNavigation from '../stats-period-navigation';
+import DatePicker from '../stats-date-picker';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import WordAdsChartTabs from '../wordads-chart-tabs';
+import titlecase from 'to-title-case';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import JetpackColophon from 'components/jetpack-colophon';
+import WordAdsEarnings from './earnings';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import PrivacyPolicyBanner from 'blocks/privacy-policy-banner';
+import StickyPanel from 'components/sticky-panel';
+
+class WordAds extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			chartTab: this.props.chartTab,
+			tabSwitched: false,
+		};
+	}
+
+	UNSAFE_componentWillReceiveProps( nextProps ) {
+		// @TODO update
+		if ( ! this.state.tabSwitched && this.state.chartTab !== nextProps.chartTab ) {
+			this.setState( {
+				tabSwitched: true,
+				chartTab: nextProps.chartTab,
+			} );
+		}
+	}
+
+	barClick = bar => {
+		this.props.recordGoogleEvent( 'Stats', 'Clicked Chart Bar' );
+		page.redirect( this.props.path + '?startDate=' + bar.data.period );
+	};
+
+	switchChart = tab => {
+		if ( ! tab.loading && tab.attr !== this.state.chartTab ) {
+			this.props.recordGoogleEvent( 'Stats', 'Clicked ' + titlecase( tab.attr ) + ' Tab' );
+			this.setState( {
+				chartTab: tab.attr,
+				tabSwitched: true,
+			} );
+		}
+	};
+
+	formatCurrency = value => {
+		return '$' + numberFormat( value, 2 );
+	};
+
+	render() {
+		const { date, site, siteId, slug, translate } = this.props;
+
+		const charts = [
+			{
+				attr: 'impressions',
+				legendOptions: [ 'impressions' ],
+				gridicon: 'visible',
+				label: translate( 'Ads Served', { context: 'noun' } ),
+			},
+			{
+				attr: 'cpm',
+				gridicon: 'stats-alt',
+				label: translate( 'Average CPM', { context: 'noun' } ),
+				format: this.formatCurrency,
+			},
+			{
+				attr: 'revenue',
+				gridicon: 'money',
+				label: translate( 'Revenue', { context: 'noun' } ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Revenue' )` *ES Score: 20*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wordads-stats)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.